providers/pg_database.rb

```diff
@@ -12,6 +12,7 @@ def whyrun_supported?
 action :create do
 
   project_name = node['enterprise']['name']
+  connection_string = "--host #{new_resource.host} --username #{new_resource.username} --password #{new_resource.password}" unless node[project_name]['postgresql']['enable']
```

Comment:
@smith I have updated the providers accordingly. I still have the admin_username and admin_password attributes on user to differentiate between the application user and the user for connecting to the database.